Support/README.md

```diff
@@ -8,9 +8,5 @@ Community members can help you on voluntary basis. See the open source and commu
 ## Commercial support
 If you'd like professional support for your Open Smart Grid Platform use case, consider support by one of the companies below.
 
```

Comment:
In the new version, there are no companies at all in the "Commercial support" section. Can an interested party contact Alliander (and provide contact details)? If so, mention this. If not, mention explicitly there's currently no commercial support.

Reply:
info@opensmartgridplatform.org would be the best way. Alliander is currenty the company answering the e-mails. This cloud change later on. I will make this clear in the text. Thanks for your input.

Reply:
Is /home/dev/Sources/SmartSocietyServices/Configuration/ based on a Vagrant script or example installation?